salt/returners/postgres_local_cache.py

```diff
@@ -232,7 +232,11 @@ def returner(load):
     sql = '''INSERT INTO salt_returns
             (fun, jid, return, id, success)
             VALUES (%s, %s, %s, %s, %s)'''
-    job_ret = {'return': six.text_type(six.text_type(load['return']), 'utf-8', 'replace')}
+    try:
+        ret = six.text_type(load['return'])
+    except UnicodeDecodeError:
+        ret = str(load['return'])
```

Comment:
Should this not use stringutils?

Reply:
@waynew This PR was merged 11 months ago. Maybe you meant to review the backport PR I opened this morning? I made the change to that PR.

Reply:
I think GitHub must have gone sideways on me. But yes, I did 😂